packages/lpa-questionnaire-web-app/tests/unit/controllers/authentication.test.js

```diff
@@ -0,0 +1,96 @@
+const authenticationController = require('../../../src/controllers/authentication');
+const { mockReq, mockRes } = require('../mocks');
+const { VIEW } = require('../../../src/lib/views');
+
+const mockEmail = 'test.email@test.uk';
+const mockLPACode = '123';
+
+const res = mockRes();
+const req = mockReq();
+req.params.lpaCode = mockLPACode;
+
+describe('authentication controller', () => {
+  describe('/:lpaCode/authentication/your-email', () => {
+    describe('show authentication enter email page', () => {
```

Comment:
If there's only one function in a describe, should it be so deeply nested?

Reply:
In fact I think this whole file could be under one describe with the description under the 'it'. Open to having my mind changed if for a good reason.

Reply:
I have removed one of the describe functions and updated the unit tests to be up to date with the latest code.